developer-extension/src/panel/components/tabs/eventsTab/eventRow.tsx

```diff
@@ -21,9 +32,33 @@ const LOG_STATUS_COLOR = {
   debug: 'cyan',
 }
 
+const RESOURCE_TYPE_LABELS: Record<string, string | undefined> = {
+  xhr: 'XHR',
+  fetch: 'Fetch',
+  document: 'Document',
+  beacon: 'Beacon',
+  css: 'CSS',
+  js: 'JS',
+  image: 'Image',
+  font: 'Font',
+  media: 'Media',
+  other: 'Other',
+}
+
 export function EventRow({ event }: { event: SdkEvent }) {
+  const [isCollapsed, setIsCollapsed] = useState(true)
+  const jsonRef = useRef<HTMLDivElement>(null)
+
   return (
-    <tr>
+    <tr
+      onClick={(event) => {
+        if (jsonRef.current?.contains(event.target as Node)) {
+          // Ignore clicks on the collapsible area
+          return
+        }
+        setIsCollapsed((previous) => !previous)
+      }}
+    >
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could we add back the pointer cursor on the event row?
users could easily miss that it is interactive otherwise

Reply:
It should be fixed